### PR TITLE
[Tests] Modifications to Support Wider Environments

### PR DESCRIPTION
**Pull request by Barakmor1**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/tests_suite_test.go [hunk 1]
```diff
@@ -178,14 +178,6 @@ func getRunningAAQ(aaqClient *clientset.Clientset) (*aaqv1.AAQ, error) {
...
 
 func updateAAQNamespaceSelector(aaqClient *clientset.Clientset, k8sClient *kubernetes.Clientset, selector *metav1.LabelSelector) error {
-	// Ensure AAQ is deployed and overwrite default namespace selector to target any namespace
-	aaqWebhook, err := k8sClient.AdmissionregistrationV1().MutatingWebhookConfigurations().Get(context.Background(), cluster.MutatingWebhookConfigurationName, metav1.GetOptions{})
-	if err != nil {
-		return err
-	}
-
-	originalWebhookSelector := aaqWebhook.Webhooks[0].NamespaceSelector
-
```
Comment (iholder101) [inline tests/tests_suite_test.go:188]: An idea for an optimization: what you can do is check if the webhook is already defined with the given selector, and if it is, return early without trying to change it.

File: tests/e2e_plugable_policies_test.go [hunk 2]
```diff
@@ -23,6 +24,9 @@ var _ = Describe("ApplicationAwareQuota plugable policies", func() {
 	Context("label-sidecar evaluator", func() {
 		ctx := context.Background()
 		BeforeEach(func() {
+			if !strings.Contains(f.KubeConfig, "application-aware-quota/_ci-configs") {
+				Skip("label-sidecar exist only in the ci environment")
+			}
```
Comment (iholder101) [inline tests/e2e_plugable_policies_test.go:29]: Is there a way to dynamically check that? This code is heavily dependent on our specific CI environment

**Review by iholder101 — COMMENTED** [review]
Looks good overall! Thank you!

File: tests/e2e_plugable_policies_test.go (hunk 2 above)
Comment (Barakmor1) [reply tests/e2e_plugable_policies_test.go]: That's a good point, thanks for noting this.

Fixed :)

File: tests/tests_suite_test.go (hunk 1 above)
Comment (Barakmor1) [reply tests/tests_suite_test.go]: Done

**Comment by iholder101** [comment]
Thank you!
/lgtm

**Pull request merged**